roles/keystone/templates/etc/apache2/sites-available/keystone.conf

```diff
@@ -32,7 +32,11 @@ Listen {{ endpoints.keystone.port.backend_api }}
 {% macro setup_oidc() -%}
     OIDCClaimPrefix "OIDC_CLAIM_"
     OIDCClaimDelimiter ";"
+    {% if ursula_os == 'ubuntu' %}
```

Comment:
seems like we can remove this if else condition and leverage - `/etc/{{ openstack_meta.apache[ursula_os].package_name }}/openidc/metadata`